eos-core-depends

```diff
@@ -13,6 +13,7 @@ btrfs-tools
 cdrdao
 cheese
 chromium-browser
+cryptsetup
```

Comment:
I think this would be more appropriate in os-depends since it has the boot integration. While it's unlikely we're going to be encrypting filesystems in the installer anytime soon, I don't see why we'd preclude it.